app/src/main/java/com/example/travenor/screen/MainActivity.kt

```diff
@@ -96,11 +104,21 @@ class MainActivity : BaseActivity() {
     private fun openHomeFragment() {
         mHomeFragment = HomeFragment.newInstance()
         supportFragmentManager.beginTransaction().addToBackStack(HomeFragment::class.java.name)
-            .replace(R.id.container_fragment, mHomeFragment).commit()
+            .replace(R.id.container_fragment, mHomeFragment!!).commit()
```

Comment:
mấy chỗ này anh nghĩ không nên force cast như này nhe em

Reply:
Vì sao anh nhỉ.
Với em thì newInstance đã đảm bảo not null nên em force cast, vị trí khác có thể null nên em khai báo global là nullable ạ

Reply:
oke em ơi vậy giữ nguyên vậy được rồi

Reply:
@gianghv-3456 
không bao giờ dc force cast e nhé,
biến nullable mình có thể set lại = null ở bất kì đâu, dự án thực tế mình làm theo team, các dev khác có thể thay đổi gây ra crash